API-3656 Replace Gotham fonts

davezuch opened the pull request:
## What does this pull request do?

We are losing our license for Gotham fonts soon, so we replace them with an open source lookalike, Metropolis.

## How should this be manually tested?

- [ ] Make sure the fonts render in UI guide

Commit:
Replace Gotham fonts

We are losing our license for Gotham fonts soon, so we replace them with
an open source lookalike, Metropolis.

citizengabe commented:
@davezuch I'm having trouble building this project using either Node versions 18 or 20. What version are you on?

citizengabe commented:
I was able to get around the webpack error I was seeing on node v18 by using a [workaround](https://github.com/webpack/webpack/issues/14532#issuecomment-947012063), but it caused an error in parcel which required me to unset the env var.

I think we need to investigate not using an old version of webpack and getting this project working on the latest version of node.

citizengabe reviewed (approved):
I worked once I got things built.

davezuch commented:
@citizengabe yeah I had to use Node 14 to get the build working. Node 14 is past EOL so in order to install it with Nix I had to run:

```sh
NIXPKGS_ALLOW_INSECURE=1 nix-shell -p nodejs_14 -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/nixos-23.05.tar.gz
```

> I think we need to investigate not using an old version of webpack and getting this project working on the latest version of node.

Yeah I wondered if we should spend time solving this instead of working around it. My feeling is that this repo is basically in maintenance mode and we won't be working with it much more, so it's not worth it. If you feel otherwise though, I'm not opposed to it being addressed.

Commit:
Update release instructions

The step of bumping the `package.json` version was only for Quicksight,
as it isn't used by PureScript package managers. Quicksight no longer
uses this library, so we can get rid of the step.

davezuch commented:
@citizengabe I pushed up one more minor change if you don't mind looking it over as well fbecd2b.

citizengabe commented:
> @citizengabe I pushed up one more minor change if you don't mind looking it over as well [fbecd2b](https://github.com/citizennet/purescript-ocelot/commit/fbecd2bf5ea9a47f99978216125e994e09fe67b6).

Looks good

davezuch commented:
Thanks for the review!

citizengabe commented:
> @citizengabe yeah I had to use Node 14 to get the build working. Node 14 is past EOL so in order to install it with Nix I had to run:
> 
> ```shell
> NIXPKGS_ALLOW_INSECURE=1 nix-shell -p nodejs_14 -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/nixos-23.05.tar.gz
> ```
> 
> > I think we need to investigate not using an old version of webpack and getting this project working on the latest version of node.
> 
> Yeah I wondered if we should spend time solving this instead of working around it. My feeling is that this repo is basically in maintenance mode and we won't be working with it much more, so it's not worth it. If you feel otherwise though, I'm not opposed to it being addressed.

I'm fine not updating as long as I can use it as a library without issue.
